### ISSUE TITLE
Screen Reader reads out "dot dot" during scene description of magnet location

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/faradays-law#209]
**Device**
Dell
**OS**
Win 10
**Browser**
Firefox
**Problem Description**
For https://github.com/phetsims/QA/issues/581. Found with NVDA on Firefox.
When reading through the scene description, it seems two periods are used at some point rather than a coma or single period. This leads to an odd sounding reading, where the description is interrupted with "dot dot" for no discernable reason.

**Visuals**
<img width="414" alt="dotdot" src="https://user-images.githubusercontent.com/41024075/101827818-ff4f9d80-3aed-11eb-9ec6-c79b638323bf.png">
https://drive.google.com/file/d/1dil6_X-zD1-TtVUnBXA1cASAOBh2yqwA/view?usp=sharing

**Comment by terracoda** [comment]
These are typos in state descriptions. It might be that there is a period in the parameter string as well as the main phrase string. Or it just might be a typo in a single string.

**Comment by terracoda** [comment]
Eventually, this description would likely be better as a single sentence (removed all instances of "the") with parameterized strings separated by a comma, rather than a period:
- "Bar magnet is at middle-right of Play Area, far from 4-loop coil."

For now, let's just get rid of the double periods.

**Comment by emily-phet** [comment]
@jessegreenberg Can you do this quick fix?

**Comment by terracoda** [comment]
The double periods might even have to do with a missing parameter in the description, not sure.

I did a quick look and found these strings starting at line 195:
 "magnetAtPositionPattern": {
      "value": "Magnet at {{position}}."
    },
    "barMagnetPositionPattern": {
      "value": "The bar magnet is {{areaPosition}}"
    },
    "barMagnetIs": {
      "value": "The bar magnet is:"
    },
    "positionOfPlayAreaPattern": {
      "value": "at the {{position}} of the Play Area."
    },

and this string at line 225:
    "magnetPositionProximityPattern": {
      "value": "{{magnetPosition}}. {{coilProximity}}."

I am not sure how or if these similarly named patterns are patched together.

**Comment by terracoda** [comment]
Please re-assign me, if my help is needed for this issue.

**Comment by jessegreenberg** [comment]
OK, I removed the punctuation from the pattern that puts the two sentences together.  https://github.com/phetsims/faradays-law/commit/f1e81bde807999d2fa6afc756db07cacf5b95d5d is the commit to move to the release branch. Removing my assignment for that part since @jbphet is moving the sim through the release process.

**Comment by jbphet** [comment]
The fix for this has been cherry-picked to the 1.4 branch.

**Comment by KatieWoe** [comment]
Checked on a11y view since we are not fully testing Screen readers in rc.2. Looks fixed there. Closing. @jbphet let me know if you want this opened or checked more in depth.

**Comment by jbphet** [comment]
> @jbphet let me know if you want this opened or checked more in depth.

Closing is fine by me since it has now been independently verified.
